sos/report/plugins/kdump.py

```diff
@@ -124,4 +124,49 @@ def setup(self):
         if self.get_option("collect-kdumps"):
             self.add_copy_spec(["/var/kdump-*"])
 
+
+class AzureKDump(KDump, AzurePlugin):
+
+    files = ('/etc/kdump.conf',)
+    packages = ('kexec-tools',)
+
+    option_list = [
+        PluginOpt("get_vm_core", default=False, val_type=bool,
```

Comment:
@pmoravec @arif-ali did we ever come up with a decision on dashes-vs-underscores for plugin options? We have a small mix today, whereas our top-level options are all dashes. I remember bringing this up on IRC a long time ago, but can't recall if we ever made a choice?

Reply:
I don't recall on any discussions, maybe it was before I got active on the project

Reply:
Yeah, the main reason is consistency for user experience. If we didn't come up with anything back then, we can start a new discussion around it and not hold up this PR.